sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientLiveTests.cs

```diff
@@ -168,6 +167,7 @@ await using (var connection = new EventHubConnection(connectionString))
         /// </summary>
         ///
         [Test]
+        [Ignore("Cannot access EventData.IsEquivalentTo in the main package.")]
```

Comment:
This is not that simple to fix. We'll probably need to move some stuff around in the shared package.